### DISCUSSION TITLE
JSON Field Type

### DISCUSSION THREAD

**Discussion by jmikrut** [payloadcms/payload#1292] [Roadmap]
We should build a JSON field type that comes with a JSON editor. Stores any type of arbitrary JSON in the database.

Marking as planned and logging here for visibility.

Implementation notes: 
- Identify existing field type to use as a reference while building out the JSON field type, i.e. the `code` field
- Take an inventory of all the places the code field is referenced/used
- Stub out a new field type that replicates the code field
- To store the actual JSON, the schema type needs to be `Schema.Types.Mixed`

**Comment by DanRibbens** [comment]
Relevant link: https://github.com/payloadcms/payload/discussions/684
